.flake8

```diff
@@ -7,7 +7,7 @@ max-line-length = 120
 # C408 ignored because we like the dict keyword argument syntax
 # E501 is not flexible enough, we're using B950 instead
 ignore =
-    E203,E305,E402,E501,E721,E741,F405,F841,F999,W503,W504,C408,E302,W291,E303,
+    E203,E305,E402,E501,E704,E721,E741,F405,F841,F999,W503,W504,C408,E302,W291,E303,
```

Comment:
I didn't see this rule under known conflicts lint rules with ruff. Also, we should probably update flake8 too as this version is a bit old and maybe we won't need to ignore it if we update it.

Reply:
We stlll should be able to enable `E704`, why is it disabled though?

Reply:
`flake8` do not allow this https://www.flake8rules.com/rules/E704.html while `ruff format` for overloaded functions violates this.

```python
if condition: foo()
def func(): ...
```